.travis.yml

```diff
@@ -12,6 +12,7 @@ services:
 env:
   global:
     - _R_CHECK_TESTS_NLINES_=0
+    - GITHUB_TOKEN=
```

Comment:
At the moment I am trying to zero out this global variable on Travis, but we should remove it from here: https://travis-ci.org/SKDE-Felles/qmongr/settings